contrib/tesseract/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "tesseract"
+pkgver = "5.4.1"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DUSE_SYSTEM_ICU=ON",
+    "-DBUILD_SHARED_LIBS=ON",
+    "-DTESSDATA_PREFIX=/usr/share",
+]
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+makedepends = [
+    "icu-devel",
+    "leptonica-devel",
+    "libarchive-devel",
+    "libcurl-devel",
+    "pango-devel",
+]
+depends = [
+    "tesseract-data-eng",
+    "tesseract-data-osd",
+]
+pkgdesc = "OCR engine"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "Apache-2.0"
+url = "https://tesseract-ocr.github.io"
+source = f"https://github.com/tesseract-ocr/tesseract/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "c4bc2a81c12a472f445b7c2fb4705a08bd643ef467f51ec84f0e148bd368051b"
+
+
+@subpackage("libtesseract")
+def _(self):
+    return self.default_libs(
+        extra=[
+            "usr/lib/libcommon_training.so",
```

Comment:
looking at the makefile used in e.g. alpine, these are never installed at all

i assume they are maybe built as .a and statically linked into libtesseract.so.5 instead?

Reply:
hm no libtesseract doesn't use those, so maybe the cli tools

Reply:
yeah, just wondering which of the things is linking them wrong

in any case they are just default `add_library`, so with shared_libs=on they are all shared, and without it libtesseract is static. i would use autotools intead here which does all this correctly

Reply:
in this case is broke :^)